Makefile

```diff
@@ -13,8 +13,8 @@ GHPAGES  = $(MODULES)/gh-pages
 
 DEST = dist
 FILE_NAME = ep
-SCRIPT_SRC = src/script/entries/ep/index.js
-STYLE_SRC = src/script/entries/ep/ep.scss
+SCRIPT_SRC = src/script/entries/ep/scripts/index.js
```

Comment:
I should have maybe been more clear in my wording within the issue. My aim was to make the structure more intuitive in the sense of extracting styles from the `src/script` folder.

Something like;
```
- src
  - script
    - index.js
  - styles
    - ep.scss
```